### PR TITLE
Add float primitives

### PR DESCRIPTION
**Pull request by acf**
Started work on the floats.   This PR really depends on #19 and the `add_more_primitives` branch

### CODE REVIEW

**Comment by iamleeg** [comment]
I'm travelling (again) and finding that the github iOS app doesn't help much with reviewing PRs. I'll look soon!

File: Interpreter.c [hunk 1]
```diff
@@ -76,6 +76,26 @@ void Interpreter_pushInteger(short integerValue) {
...
+    Interpreter_push(ObjectMemory_floatObjectOf(floatValue));
+    Interpreter_success_(YES);
+}
+
+/** convenience mechanisms for Float arithmetic, not specified in the Blue Book */
+float Interpreter_popFloat(void)
+{
+	unsigned int intValue = 0;
+	Word firstWord;
+    Word secondWord;
```
Comment (iamleeg) [inline Interpreter.c]: These variables are unused.

File: tests.h [hunk 2]
```diff
@@ -26,7 +26,7 @@ struct TestResult {
   t(&failed); \
   if (failed == 0) { \
     tr->passed++; \
-    printf("PASS " #t "\n"); \
+    /*printf("PASS " #t "\n");*/ \
```
Comment (iamleeg) [inline tests.h]: I like having these lines printed so I know that the tests I've written are actually running :)

**Review by iamleeg — CHANGES_REQUESTED** [review]
All looks good, just a couple of small changes. Nice collection of tests.

File: Interpreter.c (hunk 1 above)
Comment (acf) [reply Interpreter.c]: Ahah left over debugging detritus.

File: tests.h (hunk 2 above)
Comment (acf) [reply tests.h]: okey dokey.  I'll keep the change locally otherwise I cant tell what failed, and you can spank me the first time I screw up :-)

**Pull request merged**